node/src/BaseClient.ts

```diff
@@ -2248,7 +2254,7 @@ export class BaseClient {
      * Notes:
      * 1. `BLPOP` is a blocking command,
      * see [Blocking Commands](https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands) for more details and best practices.
-     * 2. When in cluster mode, all `keys` must map to the same `hash slot`.
+     * 2. When in cluster mode, all `keys` must map to the same hash slot.
```

Comment:
nit: in other places the cluster mode remark is listed before the blocking command remark, should we be consistent with the ordering or doesn't really matter?

Reply:
Fixed